manual_test_websocket/handler.js

```diff
@@ -83,10 +95,12 @@ module.exports.deleteListener = async (event, context) => {
 const newAWSApiGatewayManagementApi=(event, context)=>{
   const endpoint=event.requestContext.domainName+'/'+event.requestContext.stage;
   const apiVersion='2018-11-29';
-  let API=context.API;
+  let API=null;
   if (!process.env.IS_OFFLINE) {
-    API = require('aws-sdk');
+    API = AWS;
```

Comment:
haven't looked at the entirety, but is this `require` needed for anything?

Reply:
The use here of aws-sdk is to create a new `ApiGatewayManagementApi({ apiVersion, endpoint })` class to be able then to call `postToConnection({ConnectionId, Data})` in order to send messages from server to ws clients.

Reply:
I believe the commented line was the same as in the README above: `require('aws-sdk/clients/apigatewaymanagementapi');`  I suppose the same applies that it is not needed (anymore).